src/dumpMetadata/controllers/dumpMetadataController.ts

```diff
@@ -65,6 +66,9 @@ export class DumpMetadataController {
       const createdId = await this.manager.createDumpMetadata(req.body);
       this.logger.log('info', `dump metadata created successfully with id: ${createdId}`);
     } catch (error) {
+      if (error instanceof DumpNameAlreadyExistsError) {
+        (error as HttpError).status = httpStatus.UNPROCESSABLE_ENTITY;
+      }
```

Comment:
if  the error is instanceof DumpNameAlreadyExistsError, the status isnt already UNPROCESSABLE_ENTITY?

Reply:
good catch, removed the property `status` from `NotFoundError` and `DumpNameAlreadyExistsError`